### PR TITLE
add forward list method

### PR DESCRIPTION
**Pull request by babyrjw**
Appium will shutdown chromedriver process before starting a webivew session. But It didn't remove all adb forwarded ports. We need a method to find out all forwarded ports. So I add a forwardList method into appium-adb.

### CODE REVIEW

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -532,6 +532,11 @@ methods.processExists = async function (processName) {
   }
 };
 
+methods.forwardList = async function () {
```
Comment (imurchie) [inline lib/tools/adb-commands.js]: I would prefer something like `getForwardList`. Semantically, `forwardPort` actually sets the port forwarding, so one would expect `forwardList` to set the list forwarding, which it does not.

File: lib/tools/adb-commands.js [hunk 2]
```diff
@@ -532,6 +532,15 @@ methods.processExists = async function (processName) {
   }
 };
 
+/**
+ * Get TCP port forwarding with adb on the device under test.
+ * @return {string} The output of the corresponding adb command. A string contains multiple forwarding lines with line break.
+ */
+methods.getForwardList = async function () {
+  log.debug(`List forwarding ports`);
+  return await this.adbExec(['forward', '--list']);
```
Comment (imurchie) [inline lib/tools/adb-commands.js]: Now that I've thought about this a bit, I think this should return an array of strings (so, the output split by newlines). This makes more sense to me.

Comment (babyrjw) [inline lib/tools/adb-commands.js]: Yes. That's better.

**Comment by imurchie** [comment]
Published in `appium-adb@2.28.0`.

**Pull request merged**